<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VesselEditorWindow</class>
 <widget class="QMainWindow" name="VesselEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vessel Creation Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableView" name="table"/>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCustomPlot" name="plot" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Vid</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="varSelect"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Met</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="areaSelect"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Harb</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="adimSelect"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionGenerate"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;Utils and Settings</string>
    </property>
    <addaction name="action_Log_Window"/>
    <addaction name="separator"/>
    <addaction name="actionRScript_location"/>
    <addaction name="actionGen_Script_location"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockLogWindow">
   <property name="windowIcon">
    <iconset resource="vesseleditor.qrc">
     <normaloff>:/icons/Rlogo.png</normaloff>:/icons/Rlogo.png</iconset>
   </property>
   <property name="windowTitle">
    <string>Log Window</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="clearLog">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="vesseleditor.qrc">
           <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="logText"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="vesseleditor.qrc">
     <normaloff>:/icons/load.png</normaloff>:/icons/load.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="icon">
    <iconset resource="vesseleditor.qrc">
     <normaloff>:/icons/setup.png</normaloff>:/icons/setup.png</iconset>
   </property>
   <property name="text">
    <string>Generate &amp;outputs...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="vesseleditor.qrc">
     <normaloff>:/icons/quit.png</normaloff>:/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_Log_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Log Window</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="vesseleditor.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
  </action>
  <action name="actionRScript_location">
   <property name="text">
    <string>RScript location...</string>
   </property>
  </action>
  <action name="actionGen_Script_location">
   <property name="text">
    <string>Gen Script location...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vesseleditor.qrc"/>
 </resources>
 <connections/>
</ui>
